frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -291,6 +295,10 @@ export type CopyDashboardRequest = {
   is_deep_copy?: boolean | null;
 };
 
+export type UpdateDashboardKeyRequest<
+  Key extends keyof UpdateDashboardRequest,
+> = Required<Pick<UpdateDashboardRequest, "id" | Key>>;
```

Comment:
nit: I thought this is about updating the "key" (e.g. the name) of the dashboard, but looks like this is for updating any fields on the dashboard. I wonder if there is a clearer name for this.

Reply:
Ahhh I see what you mean. Something like `UpdateDashboardPropertyRequest`?